coldfront/core/user/views_/request_hub_views.py

```diff
@@ -0,0 +1,395 @@
+                *args).order_by(
+                'modified')
+
+        savio_proj_request_object.num = self.paginators
+        savio_proj_request_object.pending_queryset = \
+            self.create_paginator(project_request_pending)
+
+        savio_proj_request_object.complete_queryset = \
+            self.create_paginator(project_request_complete)
+
+        savio_proj_request_object.num_pending = project_request_pending.count()
+
+        savio_proj_request_object.title = 'Savio Project Requests'
+        savio_proj_request_object.table = \
+            'project/project_request/savio/project_request_list_table.html'
+        savio_proj_request_object.button_path = \
+            'savio-project-pending-request-list'
+        savio_proj_request_object.button_text = \
+            'Go To Savio Project Requests Main Page'
+        savio_proj_request_object.id = 'savio_project_request_section'
+
+        return savio_proj_request_object
+
+    def get_vector_project_request(self):
+        """Populates a RequestListItem with data for vector project requests"""
+        vector_proj_request_object = RequestListItem()
+        user = self.request.user
+
+        args = []
+        if not self.show_all_requests:
+            args.append(Q(pi=user) | Q(requester=user))
+
+        project_request_pending = \
+            VectorProjectAllocationRequest.objects.filter(
+                status__name__in=['Under Review', 'Approved - Processing'],
+                *args).order_by(
+                'modified')
+
+        project_request_complete = \
+            VectorProjectAllocationRequest.objects.filter(
+                status__name__in=['Approved - Complete', 'Denied'],
+                *args).order_by(
+                'modified')
+
+        vector_proj_request_object.num = self.paginators
+        vector_proj_request_object.pending_queryset = \
+            self.create_paginator(project_request_pending)
+
+        vector_proj_request_object.complete_queryset = \
+            self.create_paginator(project_request_complete)
+
+        vector_proj_request_object.num_pending = project_request_pending.count()
+
+        vector_proj_request_object.title = 'Vector Project Requests'
+        vector_proj_request_object.table = \
+            'project/project_request/vector/project_request_list_table.html'
+        vector_proj_request_object.button_path = \
+            'vector-project-pending-request-list'
+        vector_proj_request_object.button_text = \
+            'Go To Vector Project Requests Main Page'
+        vector_proj_request_object.id = 'vector_project_request_section'
+
+        return vector_proj_request_object
+
+    def get_project_join_request(self):
+        """Populates a RequestListItem with data for project join requests"""
+        proj_join_request_object = RequestListItem()
+        user = self.request.user
+
+        args = []
+        if not self.show_all_requests:
```

Comment:
Is this also showing others' requests to join my projects, if I'm a PI/manager? I think we just want to show my requests to join projects.

For each request type, it might be worth adding a brief comment to explain which requests are shown when `show_all_requests` is `False`. (The others look fine, logic-wise.)

Reply:
That makes sense. I added some help text in the form of popovers next to each title